<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RingDialog</class>
 <widget class="QDialog" name="RingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подать звонок</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/autoring.ico</normaloff>:/images/autoring.ico</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Количество подаваемых звонков</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="ringQuantity">
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Длительность одного звонка</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="ringDuration"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Продолжительность паузы между звонками</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="pauseDuration"/>
   </item>
   <item row="9" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ringQuantity</tabstop>
  <tabstop>ringDuration</tabstop>
  <tabstop>pauseDuration</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ringDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RingDialog</receiver>
   <slot>durationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RingDialog</receiver>
   <slot>pauseChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ringQuantity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RingDialog</receiver>
   <slot>quantityChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>quantityChanged(int)</slot>
  <slot>durationChanged(int)</slot>
  <slot>pauseChanged(int)</slot>
 </slots>
</ui>
